distribution/lib/Standard/Table/0.0.0-dev/src/Headers.enso

```diff
@@ -0,0 +1,19 @@
+from Standard.Base import all
+
+type Headers
```

Comment:
Shall the type itself have some note as well?

I don't remember if the CB allows displaying those.